flake.nix

```diff
@@ -22,15 +22,13 @@
         forAllSystems (
           system:
           {
-            nixpkgs-fmt = (
-              import nixpkgs {
-                inherit system;
-                overlays = [
-                  naerskOverlay
-                  self.overlay
-                ];
-              }
-            ).nixpkgs-fmt;
+            nixpkgs-fmt = (import nixpkgs {
```

Comment:
does this mean that the formatter has collapsed these lines?

Reply:
No, It shouldn't. the initial format of the flake.nix before I start contribute is:
```
        packages = forAllSystems (
          system:
            {
              nixpkgs-fmt = (
                import nixpkgs {
                  inherit system;
                  overlays = [
                    naerskOverlay
                    self.overlay
                  ];
                }
              ).nixpkgs-fmt;
            }
        );
```

and so with this PR, it will still be like the above code.

Reply:
Do you want me to revert it to the first format?

Reply:
It's alright. I just wanted to make sure that it doesn't collapse lines.